src/python/pants/backend/helm/subsystems/helm.py

```diff
@@ -165,7 +164,7 @@ class HelmSubsystem(TemplatedExternalTool):
     )
 
     args = ArgsListOption(
-        example="--dry-run",
+        example="--force",
```

Comment:
I think we should also mention the correct way to pass `--dry-run`, something like:
> use the `--experimental-deploy-dry-run` flag to pass `--dry-run`

Reply:
Good catch. I added it in another place, but I think that was an error message. Added it to this description as well.